docs/snowflake-native-apps/linkedin-ads.md

```diff
@@ -37,23 +40,23 @@ In order for the LinkedIn Ads Connector by Airbyte to query LinkedIn, you will n
 ## Snowflake Native App Authorizations
 
 :::note
-By default the app will be installed using the name `AIRBYTE_LINKEDIN_ADS`, but if you renamed the app during installation, you will have to use that name as a reference.
```

Comment:
I already changed the name of this in our provider account to be consistent with Facebook